common/CMakeLists.txt

```diff
@@ -5,9 +5,11 @@ AUX_SOURCE_DIRECTORY(./ COMMON_FILES)
 
 add_library(common ${COMMON_FILES})
 
-target_link_libraries(common jemalloc glog gflags ssl wangle folly aws-cpp-sdk-core aws-cpp-sdk-s3 boost_system boost_filesystem thrift thriftcpp2 jsoncpp_lib_static zstd)
+target_link_libraries(common jemalloc glog gflags ssl wangle folly aws-cpp-sdk-core aws-cpp-sdk-s3 boost_system boost_filesystem thrift thriftcpp2 jsoncpp_lib_static zstd rdkafka++)
```

Comment:
can we add a CMakeLists.txt in common/kafka/ and only link rdkafka++ there?

Reply:
Done. Turns out since we are called only from tests, rdkafka++ was needed only there.